Dockerfile

```diff
@@ -1,4 +1,30 @@
-FROM jakechampion/heroku-nodejs
-RUN gem install scss-lint
+# Inherit from Heroku's stack
+FROM heroku/cedar:14
+
+# Internally, we arbitrarily use port 3000
+ENV PORT 3000
+# Which version of node?
+ENV NODE_ENGINE 5.1.0
+# Locate our binaries
+ENV PATH /app/heroku/node/bin/:/app/user/node_modules/.bin:$PATH
+
+# Create some needed directories
+RUN mkdir -p /app/heroku/node /app/.profile.d
+WORKDIR /app/user
+
+# Install node
+RUN curl -s https://s3pository.heroku.com/node/v$NODE_ENGINE/node-v$NODE_ENGINE-linux-x64.tar.gz | tar --strip-components=1 -xz -C /app/heroku/node
+
+# Export the node path in .profile.d
+RUN echo "export PATH=\"/app/heroku/node/bin:/app/user/node_modules/.bin:\$PATH\"" > /app/.profile.d/nodejs.sh
+
+ADD package.json /app/user/
+
+RUN npm install --production
+RUN npm install origami-build-tools@^4.3.2
```

Comment:
This is required to compile the SASS and JS for the client. Not sure whether it should be added to the application dependencies since it is a build-time requirement.